### DISCUSSION TITLE
Scanner Plugin

### DISCUSSION THREAD

**Discussion by hamrad88** [AlexandreRouma/SDRPlusPlus#492] [Ideas]
I am just starting to use this great application and was wondering if there is a scanner plugin for it.

Thanks,

Tom

**Comment by the-gobleo-tinkerer** [comment]
A scanner plugin is already being worked on. See #299.

**Comment by hamrad88** [comment]
Excellent

 

Tom Stiles

**Comment by skideric** [comment]
FANTASTIC! Will be on the look out for the release!

**Comment by Pytlicek** [comment]
I'm a macOS user and I'm trying to get to work with SDR++ with plugins without luck.
Could you please help me with any information on how to make plugins work under macOS?
Are there any docs on this or something? Many thanks 🙏

**Comment by UT4UJS** [comment]
The plugin is missing:
1. Blocking spurious frequencies - the scanner passes the spurious frequency. Blocking is convenient to carry out when stopping at a parasitic frequency. Manual mode and automatic if stopped at the same frequency for more than XXX seconds.
2. Visual display of the level bar on the spectrum - when the plug-in is activated.
3. Creation of templates with indication of range, deviation, pitch, modulation type, squelch level for each template. Ability to select several templates for scanning.
4. Ability to activate a new bar-window at the start of the scanning module.

Very cool thing scanning! very lacking!!!

**Reply by AlexandreRouma** [reply]
As I said everywhere, it's a work in progress, writing that plugin isn't my full time job...

**Reply by UT4UJS** [reply]
Thank you ! I understand and really appreciate your work! what has been done is greatly expands the possibilities of the program!

**Comment by gbeyer3** [comment]
Holy moly! Alexandre has created an amazing SDR, basically out of the goodness of his heart, gives it away for free, and people take shots at it for what is missing.  ¯\_(ツ)_/¯

**Comment by andreitm72** [comment]
Exactly as UT4UJS says here I tried to build it myself, obviously I got stuck at 93%. Yes, these features are really needed.

**Reply by arun1969** [reply]
andreitm72 can you post full code, i can help you to improve

**Reply by andreitm72** [reply]
Hello. Thanks , I abandoned the project for technical reasons. Sdr++ is not so stable in signal analysis. If you want, you can take over from 0, I will follow your work with interest. Sincere greetings.

**Reply by arun1969** [reply]
let me know, where i can access the your work of scanner plugin source code

**Reply by arun1969** [reply]
@andreitm72 
Waiting for your positive response.
